src/java.base/share/classes/com/sun/crypto/provider/HmacCore.java

```diff
@@ -87,8 +83,7 @@ abstract class HmacCore extends MacSpi implements Cloneable {
                                 break;
                             }
                         }
-                    } catch (NoSuchAlgorithmException nsae) {
-                        continue;
+                    } catch (NoSuchAlgorithmException ignored) {
```

Comment:
Debatable whether it is ok to remove that line. If code is ever added after this line and within this for block, then this code could behave differently because it would not be the same as continue.